qa-tests-backend/src/test/groovy/SACTest.groovy

```diff
@@ -73,6 +76,22 @@ class SACTest extends BaseSpecification {
                 WAIT_FOR_VIOLATION_TIMEOUT)
         assert waitForViolation(DEPLOYMENT_QA2.name, "Secure Shell (ssh) Port Exposed",
                 WAIT_FOR_VIOLATION_TIMEOUT)
+
+        // Make sure each deployment has a risk score.
+        def deployments = DeploymentService.listDeployments()
+        deployments.each {
+            def depID = it.id
+            try {
+                withRetry(30, 2) {
+                    assert DeploymentService.getDeploymentWithRisk(depID).hasRisk()
+                }
+            } catch (Exception e) {
```

Comment:
Do we need this catch? If getDeployment fails there is huge chance that test will fail but with hard to understand reason.